apps/app/src/views/routes/Settings/components/users/UserItemActions.tsx

```diff
@@ -63,6 +83,54 @@ export const UserItemActions: React.FC<UserItemActionsProps> = ({ user }) => {
           <MenuItem icon={<FiEdit fontSize="1.2em" />} onClick={onOpen}>
             Edit Roles
           </MenuItem>
+          <MenuItem
+            icon={<FiTrash fontSize="1.3em" color="red" />}
+            disabled={deleteLoading}
+            onClick={async () => {
+              const decision = await confirmWrapper('Remove user?', {
+                description: <Text mb={2}>This will remove user from organization.</Text>,
+                cancelText: 'Cancel',
+                confirmText: 'Delete',
+                colorScheme: 'red'
+              });
+              if (decision) {
+                removeUser({
+                  variables: {
+                    userId: userData.id
+                  }
+                });
+                toast({
+                  position: 'top',
+                  // TODO: Override default solid theme with this outline variant
+                  render: ({ onClose }) => (
```

Comment:
I _think_ that Paul (@captDaylight) made a new toast component that has consistent styling

Reply:
Its called `StyledToast`

Reply:
done